### PR TITLE
fix post data with tag

### PR DESCRIPTION
**Pull request by wonderflow**
## Changes
   
  - [ ] feature1
  - [ ] feature2 
  - [ ] fixbug1
  - [ ] fixbug2
 
## Reviewers

  - [ ] @[someone] please review
  - [ ] @[someotherone] please review
    
## Checklist
   
   - [ ] Rebased/mergable
   - [ ] Tests pass
   - [ ] CHANGELOG.md updated
   - [ ] Jira issue/task done

### CODE REVIEW

**Comment by unknwon** [comment]
lgtm

File: pipeline/service.go [hunk 1]
```diff
@@ -136,7 +136,11 @@ func (c *Pipeline) NewOperation(opName string, args ...interface{}) *request.Ope
...
 	case base.OpPostData:
-		method, urlTmpl = base.MethodPost, "/v2/repos/%s/data?tags=%s"
+		if len(args) == 3 {
+			method, urlTmpl = base.MethodPost, "/v2/repos/%s/data?tags=%s"
+		} else {
+			method, urlTmpl = base.MethodPost, "/v2/repos/%s/data"
+		}
 	case base.OpPostTextData:
 		method, urlTmpl = base.MethodPost, "/v2/streams/%s/data?tags=%s&rules=%s"
 	case base.OpPostRawtextData:
```
Comment (redHJ) [inline pipeline/service.go]: OpPostRawtextData也需要兼容

**Comment by redHJ** [comment]
lgtm

**Comment by unknwon** [comment]
LGTM 2

**Pull request merged**